.templates/deconz/service.yml

```diff
@@ -0,0 +1,26 @@
+  deconz:
+    image: marthoc/deconz
+    container_name: deconz
+    restart: unless-stopped
+    network_mode: bridge
+    ports:
+      - '80:80'
+      - '443:443'
+      - '5901:5900'
+    volumes:
+       - ./volumes/deconz/:/root/.local/share/dresden-elektronik/deCONZ
+    devices:
+      #ConBee II:
+      - /dev/ttyACM0
```

Comment:
This needs to be commented out so that an error doesn't occur when someone tries to run the docker up command by default.